themes/barbuk/barbuk.theme.bash

```diff
@@ -88,7 +88,14 @@ function _prompt {
         ssh_info="${bold_blue}\u${bold_orange}@${cyan}$host ${bold_orange}in"
     fi
 
-    PS1="\\n${ssh_info} ${purple}$(scm_char)${dir_color}\\w${normal}$(scm_prompt_info)${exit_code}"
+    # Detect python venv
+    if [[ -n "${CONDA_DEFAULT_ENV}" ]]; then
+        python_venv="${CONDA_DEFAULT_ENV}"
+    elif [[ -n "${VIRTUAL_ENV}" ]]; then
+        python_venv=$(basename "${VIRTUAL_ENV}")
+    fi
+
+    PS1="\\n${ssh_info} ${python_venv} ${purple}$(scm_char)${dir_color}\\w${normal}$(scm_prompt_info)${exit_code}"
```

Comment:
This looks good - my only concern is that this adds the Python venv for all users of this theme. This might not be an issue, but some people might complain about the (possibly unwanted) additional info.

I'm fine with keeping it like this, but we might need to make it optional if people don't like this change...

Reply:
Seeing as it only appears within the prompt when in an activated python virtual environment, maybe a good compromise would be making it optional, but enabled by default?

Reply:
I think it's OK to leave it as is - if people are requesting it to be optional, we can still change that later.